### PR TITLE
python sub-version check, pyarrow check

### PR DESCRIPTION
**Pull request by YuechengLiu**

### CODE REVIEW

**Comment by zenghsh3** [comment]
fixed #446

**Review by zenghsh3 — DISMISSED** [review]
LGTM

**Comment by YuechengLiu** [comment]
### Reason for "pyarrow" check
To make sure that 'master', 'worker' and 'client' hold the same serialization method.

File: parl/remote/worker.py [hunk 1]
```diff
@@ -76,7 +77,7 @@ def __init__(self, master_address, cpu_num=None, log_server_port=None):
         self._set_cpu_num(cpu_num)
         self.job_buffer = queue.Queue(maxsize=self.cpu_num)
         self._create_sockets()
-        self.check_version()
+        self.check_version_and_package()
```
Comment (TomorrowIsAnOtherDay) [inline parl/remote/worker.py]: Please rename the function to `check_env_consistency`. Same modification suggestion at the client.py.

File: parl/remote/utils.py [hunk 2]
```diff
@@ -191,3 +191,19 @@ def get_subfiles_recursively(folder_path):
             elif len(dirs) == 0:
                 empty_subfolders.append(os.path.normpath(root))
         return python_files, other_files, empty_subfolders
+
+
+def has_module(module_name):
+    '''
```
Comment (TomorrowIsAnOtherDay) [inline parl/remote/utils.py]: add the comment: 
Check if the python environment has installed the module or package.

File: parl/remote/worker.py (hunk 1 above)
Comment (YuechengLiu) [reply parl/remote/worker.py]: fix

File: parl/remote/utils.py (hunk 2 above)
Comment (YuechengLiu) [reply parl/remote/utils.py]: fixed

**Review by TomorrowIsAnOtherDay — DISMISSED** [review]
LGTM.

**Review by TomorrowIsAnOtherDay — APPROVED** [review]

**Pull request merged**